examples/project/main.tf

```diff
@@ -25,10 +25,40 @@ resource "vra_project" "this" {
 
   administrators = ["jason@vra.local"]
 
+  administrator {
+    email = "jason@vra.local"
+    type = "user"
+  }
+
+  administrator {
```

Comment:
should we make this administrator_list instead?

Reply:
Same as above